ci-operator/step-registry/ipi/conf/aws/blackholenetwork/ipi-conf-aws-blackholenetwork-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# TODO: move to image
+curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/install-config-blackholenetwork.yaml.patch"
+
+aws_region=$(/tmp/yq r "${CONFIG}" 'platform.aws.region')
+
+subnets="[]"
+case "${aws_region}_$((RANDOM % 4))" in
```

Comment:
No need for `RANDOM` here yet.  Ir if you do keep it, use `$((RANDOM % 1))`.  Because there's currently only the single choice within each region.